en/homepage/modes_statically_generated.md

```diff
@@ -3,7 +3,7 @@ title: Statically Generated
 content_title: Statically Generated
 ---              
 Static Site Generation is a very hot topic right now (aka JAMStack). Instead of switching to another framework and
-spending time to get used to it, why not kill two birds with one stone?
+spending time to get used to it, why not killing two birds with one stone?
```

Comment:
why? Dont think its supposed to be in this PR or right change at all.

Reply:
fixed. some merge errors